lib/stacks/infra/core.ts

```diff
@@ -435,6 +454,31 @@ export class CoreStack extends Stack {
             policy: AwsCustomResourcePolicy.fromSdkCalls({ resources: [appConfigTable.tableArn] }),
         });
 
+        new AwsCustomResource(this, 'populate-instance-types', {
+            onCreate: {
+                service: 'Lambda',
+                action: 'invoke',
+                physicalResourceId: PhysicalResourceId.of('initInstanceTypes'),
+                parameters: {
+                    FunctionName: appConfigLambda.functionName,
+                    Payload: '{}'
+                }, 
+            },
+            policy: AwsCustomResourcePolicy.fromStatements([
```

Comment:
Like discussed in the Dev Sync, this will fail on customer deployments because their roles aren't allowed to create IAM resources. I think that the App Role would work for this and that is already created.

Reply:
okay good idea, will try giving it the app role